doc/about/about.rst

```diff
@@ -1,72 +1,108 @@
+How can I cite Binder?
+----------------------
 
-How much does running ``mybinder.org`` cost?
---------------------------------------------
+If you publish work that uses Binder, please consider citing the
+`Binder paper from the 2018 SciPy proceedings <http://conference.scipy.org/proceedings/scipy2018/project_jupyter.html>`_!
 
-Great question! If you're interested in the technical costs of running
-``mybinder.org``, we publish a semi-up-to-date dataset of our costs at the
-`binder-data <https://github.com/jupyterhub/binder-data/tree/master/billing/data/proc>`_
-repository. In addition, you can explore these costs with the binder link below!
+Here is a citation that you can use:
 
-.. image:: https://mybinder.org/badge_logo.svg
-   :target: https://mybinder.org/v2/gh/jupyterhub/binder-billing/master?urlpath=lab/tree/analyze_data.ipynb
+.. code-block:: raw
 
-How can ``mybinder.org`` be free to use?
-----------------------------------------
+   Jupyter et al., "Binder 2.0 - Reproducible, Interactive, Sharable
+   Environments for Science at Scale." Proceedings of the 17th Python
+   in Science Conference. 2018. doi://10.25080/Majora-4af1f417-011
 
-See :ref:`about` for more information on the mybinder.org team and who provides
-the resources to pay for the service. Generally, mybinder.org is run with modest resources
-provided to users in order to keep costs down. In the future we hope to see more
-public BinderHub services running that can form a collection of community
-resources for interactive cloud computing.
+https://doi.org/10.25080/Majora-4af1f417-011
+
+
+Who runs mybinder.org?
+----------------------
+
+The service at ``mybinder.org`` is powered by a collection of BinderHub
+deployments called :doc:`The BinderHub Federation <federation>` and run by
+the `Binder team <https://jupyterhub-team-compass.readthedocs.io/en/latest/team.html#binder-team>`_.
+
+For more information about the BinderHubs behind ``mybinder.org``, see
+:doc:`The BinderHub Federation <federation>`.
+
+Who pays for mybinder.org?
+--------------------------
+
+The service at mybinder.org is provided by a federation of BinderHub deployments
+that are maintained by several members and organizations in the Binder community.
+For information about who runs and pays for these deployments, see {ref}`about/federation`.
+
+Is there dedicated funding for the Binder Project?
+--------------------------------------------------
+
+The Binder Project currently has no dedicated funding (beyond the time and hardware costs
+provided by its {ref}`federation members <about/federation>`).
+
+In 2017 the Binder Project received a 1-year grant from the
+`Moore Foundation <https://figshare.com/s/e9d0ad7bdc4e405cccfa>`_.
+
+What technology runs mybinder.org?
+----------------------------------
+
+The technology behind mybinder.org is primarily composed of three open-source projects:
+
+* `JupyterHub <https://z2jh.jupyter.org>`_, which manages cloud infrastructure for user instances
```

Comment:
Since we're stating that this is managing cloud infrastructure specifically, should we link to z2jh-k8s?

Reply:
Which link would you change? Isn't this already linking to that?

Reply:
Yes sorry, my mistake!